public/app/features/search/components/SearchResultsFilter.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { FC } from 'react';
+import { css } from 'emotion';
+import { Button, Forms, stylesFactory, useTheme } from '@grafana/ui';
+import { GrafanaTheme, SelectableValue } from '@grafana/data';
+
+type onSelectChange = (value: SelectableValue) => void;
+
+export interface Props {
+  allChecked?: boolean;
+  canDelete?: boolean;
+  canMove?: boolean;
+  deleteItem: () => void;
+  moveTo: () => void;
+  onSelectAllChanged: any;
+  onStarredFilterChange: onSelectChange;
+  onTagFilterChange: onSelectChange;
+  selectedStarredFilter: string;
+  selectedTagFilter: string;
+  tagFilterOptions: SelectableValue[];
+}
+
+const starredFilterOptions = [
+  { label: 'Yes', value: true },
+  { label: 'No', value: false },
+];
+
+export const SearchResultsFilter: FC<Props> = ({
+  allChecked,
+  canDelete,
+  canMove,
+  deleteItem,
+  moveTo,
+  onSelectAllChanged,
+  onStarredFilterChange,
+  onTagFilterChange,
+  selectedStarredFilter,
+  selectedTagFilter,
+  tagFilterOptions,
+}) => {
+  const showActions = canDelete || canMove;
+  const theme = useTheme();
+  const styles = getStyles(theme);
+
+  return (
+    <div className={styles.wrapper}>
+      <Forms.Checkbox value={allChecked} onChange={onSelectAllChanged} />
+      <div className={styles.row}>
+        {showActions ? (
+          <div className={styles.row}>
+            <Button disabled={!canMove} onClick={moveTo} icon="fa fa-exchange" variant="secondary">
+              Move
+            </Button>
+            <Button disabled={!canDelete} onClick={deleteItem} icon="fa fa-trash" variant="destructive">
+              Delete
+            </Button>
+          </div>
+        ) : (
+          <div className={styles.row}>
+            <Forms.Select
+              size="sm"
+              placeholder="Filter by starred"
+              key={selectedStarredFilter}
+              options={starredFilterOptions}
+              onChange={onStarredFilterChange}
+            />
+
+            <Forms.Select
+              size="sm"
+              placeholder="Filter by tag"
+              key={selectedTagFilter}
+              options={tagFilterOptions}
+              onChange={onTagFilterChange}
+            />
+          </div>
```

Comment:
Can you use `HorizontalLayout` component here to wrap the selects? It will provide some spacing between the selects which are now next to each other without any spacing.

Reply:
I guess you mean `HorizontalGroup` from grafana/ui? I'll try it out.

Reply:
Yes, the one from ui